src/pane-element.coffee

```diff
@@ -71,6 +71,8 @@ class PaneElement extends HTMLElement
     hasFocus = @hasFocus()
     itemView = atom.views.getView(item)
 
+    if item.getPath? then $(itemView).addClass('has-file-path')
```

Comment:
It'd be nice to use the native DOM API rather than jQuery here, so that we can eventually remove our jQuery dependency entirely. Also, we usually use the postfix `if` for these one-liners.

```coffee
itemView.classList.add('has-file-path') if item.getPath?
```